src/components/Markdown.tsx

```diff
@@ -44,6 +44,19 @@ export default function Markdown({
       notebook.insertCodeBelowActiveCell &&
       (language || '').toLowerCase().startsWith('py');
 
+    const isSingleLine =
+      typeof codeString === 'string' &&
+      !codeString.includes('\n') &&
+      codeString.length <= 20;

```

Comment:
we should probably refactor the 20 magic variable into a constant with a short comment above explaining why 20 is a good cutoff